<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="XWJPMDD177" name="app.psc.testcases.myprofile.MyProfileTestCases" tests="9" failures="9" timestamp="17 Nov 2017 09:57:07 GMT" time="0.906" errors="0">
  <testcase name="TC002_02_SIT_PSC_Update_Email_Address_Negative_Flow" time="0.031" classname="app.psc.testcases.myprofile.MyProfileTestCases">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at core.framework.TestListener.isFinalTestStatus(TestListener.java:78)
	at core.framework.TestListener.afterInvocation(TestListener.java:332)
	at org.testng.internal.invokers.InvokedMethodListenerInvoker$InvokeAfterInvocationWithoutContextStrategy.callMethod(InvokedMethodListenerInvoker.java:100)
	at org.testng.internal.invokers.InvokedMethodListenerInvoker.invokeListener(InvokedMethodListenerInvoker.java:62)
	at org.testng.internal.Invoker.runInvokedMethodListeners(Invoker.java:619)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:777)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
    </failure>
  </testcase> <!-- TC002_02_SIT_PSC_Update_Email_Address_Negative_Flow -->
  <testcase name="TC004_01_SIT_PSC_Update_My_profile_Security_Questions" time="0.063" classname="app.psc.testcases.myprofile.MyProfileTestCases">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at core.framework.TestListener.isFinalTestStatus(TestListener.java:78)
	at core.framework.TestListener.afterInvocation(TestListener.java:332)
	at org.testng.internal.invokers.InvokedMethodListenerInvoker$InvokeAfterInvocationWithoutContextStrategy.callMethod(InvokedMethodListenerInvoker.java:100)
	at org.testng.internal.invokers.InvokedMethodListenerInvoker.invokeListener(InvokedMethodListenerInvoker.java:62)
	at org.testng.internal.Invoker.runInvokedMethodListeners(Invoker.java:619)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:777)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
    </failure>
  </testcase> <!-- TC004_01_SIT_PSC_Update_My_profile_Security_Questions -->
  <testcase name="TC002_01_SIT_PSC_Update_Email_Address_Positive_Flow" time="0.016" classname="app.psc.testcases.myprofile.MyProfileTestCases">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at core.framework.TestListener.isFinalTestStatus(TestListener.java:78)
	at core.framework.TestListener.afterInvocation(TestListener.java:332)
	at org.testng.internal.invokers.InvokedMethodListenerInvoker$InvokeAfterInvocationWithoutContextStrategy.callMethod(InvokedMethodListenerInvoker.java:100)
	at org.testng.internal.invokers.InvokedMethodListenerInvoker.invokeListener(InvokedMethodListenerInvoker.java:62)
	at org.testng.internal.Invoker.runInvokedMethodListeners(Invoker.java:619)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:777)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
    </failure>
  </testcase> <!-- TC002_01_SIT_PSC_Update_Email_Address_Positive_Flow -->
  <testcase name="TC001_02_SIT_PSC_Update_My_profile_Default_Plan" time="0.032" classname="app.psc.testcases.myprofile.MyProfileTestCases">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at core.framework.TestListener.isFinalTestStatus(TestListener.java:78)
	at core.framework.TestListener.afterInvocation(TestListener.java:332)
	at org.testng.internal.invokers.InvokedMethodListenerInvoker$InvokeAfterInvocationWithoutContextStrategy.callMethod(InvokedMethodListenerInvoker.java:100)
	at org.testng.internal.invokers.InvokedMethodListenerInvoker.invokeListener(InvokedMethodListenerInvoker.java:62)
	at org.testng.internal.Invoker.runInvokedMethodListeners(Invoker.java:619)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:777)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
    </failure>
  </testcase> <!-- TC001_02_SIT_PSC_Update_My_profile_Default_Plan -->
  <testcase name="TC005_01_SIT_PSC_Update_My_profile_Password" time="0.031" classname="app.psc.testcases.myprofile.MyProfileTestCases">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at core.framework.TestListener.isFinalTestStatus(TestListener.java:78)
	at core.framework.TestListener.afterInvocation(TestListener.java:332)
	at org.testng.internal.invokers.InvokedMethodListenerInvoker$InvokeAfterInvocationWithoutContextStrategy.callMethod(InvokedMethodListenerInvoker.java:100)
	at org.testng.internal.invokers.InvokedMethodListenerInvoker.invokeListener(InvokedMethodListenerInvoker.java:62)
	at org.testng.internal.Invoker.runInvokedMethodListeners(Invoker.java:619)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:777)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
    </failure>
  </testcase> <!-- TC005_01_SIT_PSC_Update_My_profile_Password -->
  <testcase name="TC006_01_FAQ_DDTC_245" time="0.047" classname="app.psc.testcases.myprofile.MyProfileTestCases">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at core.framework.TestListener.isFinalTestStatus(TestListener.java:78)
	at core.framework.TestListener.afterInvocation(TestListener.java:332)
	at org.testng.internal.invokers.InvokedMethodListenerInvoker$InvokeAfterInvocationWithoutContextStrategy.callMethod(InvokedMethodListenerInvoker.java:100)
	at org.testng.internal.invokers.InvokedMethodListenerInvoker.invokeListener(InvokedMethodListenerInvoker.java:62)
	at org.testng.internal.Invoker.runInvokedMethodListeners(Invoker.java:619)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:777)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
    </failure>
  </testcase> <!-- TC006_01_FAQ_DDTC_245 -->
  <testcase name="TC001_01_SIT_PSC_Update_My_profile_Default_Plan" time="0.624" classname="app.psc.testcases.myprofile.MyProfileTestCases">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at core.framework.TestListener.isFinalTestStatus(TestListener.java:78)
	at core.framework.TestListener.afterInvocation(TestListener.java:332)
	at org.testng.internal.invokers.InvokedMethodListenerInvoker$InvokeAfterInvocationWithoutContextStrategy.callMethod(InvokedMethodListenerInvoker.java:100)
	at org.testng.internal.invokers.InvokedMethodListenerInvoker.invokeListener(InvokedMethodListenerInvoker.java:62)
	at org.testng.internal.Invoker.runInvokedMethodListeners(Invoker.java:619)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:777)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
    </failure>
  </testcase> <!-- TC001_01_SIT_PSC_Update_My_profile_Default_Plan -->
  <testcase name="TC003_01_SIT_PSC_Update_User_Name" time="0.031" classname="app.psc.testcases.myprofile.MyProfileTestCases">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at core.framework.TestListener.isFinalTestStatus(TestListener.java:78)
	at core.framework.TestListener.afterInvocation(TestListener.java:332)
	at org.testng.internal.invokers.InvokedMethodListenerInvoker$InvokeAfterInvocationWithoutContextStrategy.callMethod(InvokedMethodListenerInvoker.java:100)
	at org.testng.internal.invokers.InvokedMethodListenerInvoker.invokeListener(InvokedMethodListenerInvoker.java:62)
	at org.testng.internal.Invoker.runInvokedMethodListeners(Invoker.java:619)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:777)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
    </failure>
  </testcase> <!-- TC003_01_SIT_PSC_Update_User_Name -->
  <testcase name="TC002_03_SIT_PSC_Update_Email_Address_Invalid_Password" time="0.031" classname="app.psc.testcases.myprofile.MyProfileTestCases">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at core.framework.TestListener.isFinalTestStatus(TestListener.java:78)
	at core.framework.TestListener.afterInvocation(TestListener.java:332)
	at org.testng.internal.invokers.InvokedMethodListenerInvoker$InvokeAfterInvocationWithoutContextStrategy.callMethod(InvokedMethodListenerInvoker.java:100)
	at org.testng.internal.invokers.InvokedMethodListenerInvoker.invokeListener(InvokedMethodListenerInvoker.java:62)
	at org.testng.internal.Invoker.runInvokedMethodListeners(Invoker.java:619)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:777)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
    </failure>
  </testcase> <!-- TC002_03_SIT_PSC_Update_Email_Address_Invalid_Password -->
</testsuite> <!-- app.psc.testcases.myprofile.MyProfileTestCases -->
